src/worker/main.ts

```diff
@@ -30,7 +30,12 @@ function measureElapsedTime(job: Queue.Job, tags) {
 
 const queueOpts: QueueOptions = {
 	defaultJobOptions: {
-		attempts: 3,
+		attempts: 5,
+		timeout: 60000,
```

Comment:
Wow, that looks too easy! :) 

Let's test it by simulating a long wait in src/config/enhance-octokit.ts:18